src/App.vue

```diff
@@ -184,6 +201,15 @@ export default {
 
 			this.setPageTitle(this.getConversationName(this.token), this.atLeastOneLastMessageIdChanged)
 		},
+
+		token() {
+			// Collapse the sidebar if it's a 1to1 conversation
+			if (this.isOneToOne || BrowserStorage.getItem('sidebarOpen') === 'false') {
```

Comment:
some day we'll store these in the vuex store and the vuex store will take care to store them in BrowserStorage